### ISSUE TITLE
Load JSON data into NestedField

### ISSUE THREAD

**Issue by larry0123du** [pytorch/text#489]
If I have a train.json and a test.json that contain JSON data of the below format:
`{"review": [["it", "is", "good"], ["I", "like", "it"]], "label": 1}`
When I was trying to use the following code to load it:
```python
import torch
from torchtext import data, datasets

REVIEW_NEST = data.Field()
REVIEW = data.NestedField(REVIEW_NEST)
LABEL = data.LabelField(dtype=torch.float)

fields = {'review': ('r', REVIEW), 'label': ('l', LABEL)}

train_data, test_data = data.TabularDataset.splits(
                            path = 'popcorn_data',
                            train = 'train.json',
                            test = 'test.json',
                            format = 'json',
                            fields = fields
)

REVIEW.build_vocab(train_data.review, max_size=25000)
print(REVIEW.vocab.freqs.most_common(20))
```
I got an empty list. Is there anything I did wrong here? Thanks!

**Comment by tu-artem** [comment]
with `fields = {'review': ('r', REVIEW), 'label': ('l', LABEL)}` you are renaming `review `to `r` and `label` to `l`. Later, in `REVIEW.build_vocab()` you should use new names, so `REVIEW.build_vocab(train_data.r, max_size=25000)` should work fine

**Comment by larry0123du** [comment]
Got it, Thanks!
